readthedocs/templates/core/project_detail_right.html

```diff
@@ -61,16 +61,25 @@ <h3>{% trans "Badge" %}</h3>
 
 
 {% block tags %}
-  {% if project.tags.count %}
     <h3>{% trans "Tags" %}</h3>
     <p>
-      {% for tag in project.tags.all %}
-        <a href="{% url "projects_tag_detail" tag.slug %}">{{ tag.name }}</a>{% if forloop.last %}{% else %}, {% endif %}
-      {% empty %}
-        <span class="quiet">{% trans "No tags" %}</span>
-      {% endfor %}
+
+    {% if project.tags.count %}
```

Comment:
I saw that we are using `.count` in some places where we don't need the number but we want to know the existence. So, maybe it's better to use `.exists`

Reply:
Hah yea, old habit!